packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab/components/ConnectorSetupWarning.tsx

```diff
@@ -1,25 +1,19 @@
-import type { ConnectorResponse } from '@logto/schemas';
 import { ConnectorType } from '@logto/schemas';
 import { useTranslation } from 'react-i18next';
-import useSWR from 'swr';
 
 import Alert from '@/components/Alert';
-import type { RequestError } from '@/hooks/use-api';
+import useEnabledConnectorTypes from '@/hooks/use-enabled-connector-types';
 
 type Props = {
   requiredConnectors: ConnectorType[];
 };
 
 const ConnectorSetupWarning = ({ requiredConnectors }: Props) => {
-  const { data: connectors } = useSWR<ConnectorResponse[], RequestError>('/api/connectors');
+  const { data: enabledConnectors } = useEnabledConnectorTypes();
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
 
-  if (!connectors) {
-    return null;
-  }
-
   const missingConnectors = requiredConnectors.filter(
-    (connectorType) => !connectors.some(({ type, enabled }) => type === connectorType && enabled)
+    (connectorType) => !enabledConnectors.includes(connectorType)
```

Comment:
nit: Should use `isConnectorTypeEnabled` instead? Or you can remove it from the hook since it's not used anyway